### PR TITLE
rajavineel adds Gajwel hometown file

### PR DESCRIPTION
**Pull request by rajavineel**
This PR will add content to Gajwel hometown file

### CODE REVIEW

File: vineel-gajwel.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This is the first file I have created.
```
Comment (keran4) [inline vineel-gajwel.md]: Congrats :+1:

Comment (mrrbab) [inline vineel-gajwel.md]: @rajavineel whats the home town attractions

Comment (rajavineel) [inline vineel-gajwel.md]: @mrrbab extreme weather conditions all the seasons!!

Comment (rajavineel) [inline vineel-gajwel.md]: @keran4 Thanks :)

**Pull request merged**